src/runtime/components/Carousel.vue

```diff
@@ -265,6 +267,8 @@ function isCarouselItem(item: CarouselItem): item is _CarouselItem {
 }
 
 onMounted(() => {
+  loadPlugins()
```

Comment:
I would keep a watch immediate on the props to load the plugins instead, it's no longer reactive.